net/modemmanager/files/25-modemmanager-net

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Copyright (C) 2016 Velocloud Inc
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+
+# Load common utilities
+. /etc/modemmanager/modemmanager.common
+
+# Always make sure the rundir exists
+/bin/mkdir -m 0755 -p "${MODEMMANAGER_RUNDIR}"
```

Comment:
/bin/mkdir? What is PATH when this script is running? Please simply use mkdir and set PATH accordingly when needed...